README.md

```diff
@@ -34,3 +34,9 @@ To build just run:
 meson build
 ninja -C build
 ```
+
```

Comment:
This change clutters the README. There are many more build options available and they are self-documenting.